src/mappers/isValueMapper.ts

```diff
@@ -0,0 +1,4 @@
+import { ValueMapper } from '../abstracts/ValueMapper';
+
+export const isValueMapper: (acceptedValues: string[]) => ValueMapper<boolean> = (acceptedValues: string[]) => value =>
```

Comment:
Could I ask you to add type above for '(acceptedValues: string[]) => ValueMapper<boolean>'?

I asked about that, because this will be much easier to understand 👍

Reply:
Sure. Really improves readability